sparkprs/models.py

```diff
@@ -189,6 +193,18 @@ def _compute_commenters(self):
                         Issue.ASKED_TO_CLOSE_REGEX.search(comment['body']) is not None)
         return sorted(res.items(), key=lambda x: x[1]['date'], reverse=True)
 
+    def refresh(self):
+        try:
+            raw_github_request(get_pulls_base() + '/%i' % self.number,
+                               oauth_token=oauth_token, etag=self.etag)
+        except Exception as e:
+            if "404" in str(e):
```

Comment:
It'd probably be safer to modify `raw_github_request` to throw a more specific exception for 404 and then catch that here. There might already be an appropriate exception in urllib.